### PR TITLE
Add pyeee

### PR DESCRIPTION
**Pull request by mcuntz**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by mcuntz** [comment]
@conda-forge/help-python The recipe has passed all checks and is ready for your review. Thanks.

File: recipes/pyeee/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  host:
+    - python >=3.8
+    - pip
+    - setuptools >=42
+    - setuptools_scm >=3.5
+  run:
+    - python >=3.8
+    - numpy
+    - scipy
+    - schwimmbad <=0.3.2
```
Comment (jakirkham) [inline recipes/pyeee/meta.yaml:27]: Curious as to why the upper bound is needed

**Review by jakirkham — COMMENTED** [review]
Thanks Matthias! 🙏

Had a question below

File: recipes/pyeee/meta.yaml (hunk 1 above)
Comment (mcuntz) [reply recipes/pyeee/meta.yaml:27]: Thanks for checking. They updated schwimmbad yesterday after 2 years and seemed to have introduced a bug (I already reported it to them). I get "UnboundLocalError: local variable 'MultiPool' referenced before assignment" when using it in the tests.

Comment (mcuntz) [reply recipes/pyeee/meta.yaml]: Adrian fixed the bug. My tests run again :-) So I removed the upper bound on schwimmbad.

Comment (jakirkham) [reply recipes/pyeee/meta.yaml]: Glad to hear it!

Maybe we should exclude the problematic version (unless it was also yanked from PyPI) just as a safety measure

File: recipes/pyeee/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  host:
+    - python >=3.8
+    - pip
+    - setuptools >=42
+    - setuptools_scm >=3.5
+  run:
+    - python >=3.8
+    - numpy
+    - scipy
+    - schwimmbad
```
Comment (jakirkham) [inline recipes/pyeee/meta.yaml:27]:
```suggestion
    - schwimmbad !=0.3.2
```

**Review by jakirkham — COMMENTED** [review]
Added a small suggestion to exclude the version that was causing issues before

Otherwise LGTM

File: recipes/pyeee/meta.yaml (hunk 2 above)
Comment (mcuntz) [reply recipes/pyeee/meta.yaml]: Good idea. Done.

**Review by jakirkham — APPROVED** [review]

**Comment by jakirkham** [comment]
Thanks Matthias! 🙏

**Pull request merged**